src/pycram/datastructures/dataclasses.py

```diff
@@ -862,11 +892,12 @@ def intersected(self) -> bool:
 @dataclass
 class MultiverseContactPoint:
```

Comment:
The thing is the clients of multiverse should be independent on pycram, and then in multiverse.py which is the integration with pycram the conversion between the datatypes is done.

Also because this makes it easier as the clients have no access to the World, they just use body names.

Reply:
One could change that and make the clients have a world

Reply:
Also there's a big breaking change that is coming for multiverse that will have to change most of the structure that is currently there.

Reply:
basically there will be no clients